apps/docs/components/Navigation/Footer.tsx

```diff
@@ -11,7 +11,9 @@ const Footer = () => {
       <hr className="border-scale-400  mt-8"></hr>
       <div className="flex gap-4 items-center mt-6 justify-between">
         <div className="flex flex-col lg:flex-row gap-3 ">
-          <span className="text-xs text-scale-900">Supabase 2022</span>
+          <Link href="https://supabase.com/">
+            <a className="text-xs text-scale-900">Supabase</a>
+          </Link>
```

Comment:
I don't think this is what @MildTomato meant. I think he meant for example on the marketing site we do `© Supabase Inc` in the footer. Attached an image below of the marketing site footer for reference.

<img width="164" alt="CleanShot 2023-04-26 at 16 38 43@2x" src="https://user-images.githubusercontent.com/70828596/234696860-0bcf106e-db0f-466a-aa2f-46477a8c167d.png">

Reply:
Okay, I did not think of that, I have just updated it.

![Screenshot 2023-04-27 at 00-34-41 Supabase Docs](https://user-images.githubusercontent.com/59878203/234709694-a3f5dc77-bd8d-494b-b07d-62ee6ad2e0a6.png)